### PR TITLE
[New] Upgrade to AndroidX

### PR DESCRIPTION
**Pull request by roninprogrammer**
Migrate support to AndroidX

# Summary

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by emilioicai** [comment]
I'm a bit reluctant to migrate to AndroidX for the time being. Right now 99% of RN libraries do not support android jetpack so adding dependency with Androidx import is just gonna break about every app.

**Comment by vicknesh-dacsee** [comment]
I believe that most of plugins in RN do support android jetpacks and even now in RN 6.0 which has been migrated over to AndroidX. Please advice

**Comment by rozPierog** [comment]
@emilioicai just release it as major version with breaking changes. If someone wants to stay on <0.60 let them use 3.X.X as long as they want. It's not like there are new additions to `lottie-react-native` that are going to make 3.X.X obselte anytime soon. Right now, this library is only one that prevents me from fully commiting to RN 0.60

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rozPierog) [inline .project]: I don't believe this file is needed

File: .settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```
Comment (rozPierog) [inline .settings/org.eclipse.buildship.core.prefs]: and the same with this

File: src/android/build.gradle [hunk 3]
```diff
@@ -5,16 +5,16 @@ def safeExtGet(prop, fallback) {
 apply plugin: 'com.android.library'
 apply from: 'gradle-maven-push.gradle'
 
-def DEFAULT_ANDROID_SUPPORT_LIB_VERSION = "27.0.2"
+//def DEFAULT_ANDROID_SUPPORT_LIB_VERSION = "27.0.2"
```
Comment (rozPierog) [inline src/android/build.gradle]: we don't need this anymore

File: src/android/build.gradle [hunk 4]
```diff
@@ -28,9 +28,10 @@ android {
 }
 
 dependencies {
-  def supportLibVersion = safeExtGet('supportLibVersion', DEFAULT_ANDROID_SUPPORT_LIB_VERSION)
+  //def supportLibVersion = safeExtGet('supportLibVersion', DEFAULT_ANDROID_SUPPORT_LIB_VERSION)
```
Comment (rozPierog) [inline src/android/build.gradle]: and this

File: src/android/build.gradle [hunk 5]
```diff
@@ -28,9 +28,10 @@ android {
...
 
   implementation "com.facebook.react:react-native:+"
-  implementation "com.android.support:appcompat-v7:$supportLibVersion"
+  //implementation "com.android.support:appcompat-v7:$supportLibVersion"
```
Comment (rozPierog) [inline src/android/build.gradle]: also this

**Comment by emilioicai** [comment]
closing this in favor of https://github.com/react-native-community/lottie-react-native/pull/515/files

**Pull request closed without merging**